.github/workflows/build-publish.yml

```diff
@@ -95,6 +107,35 @@ jobs:
           subject-name: ${{ env.ECR_HOSTNAME }}/${{ needs.checks.outputs.ecr-image-name }}
           push-to-registry: true
 
+  # XXX: This is using a new way to build images. Other jobs should be updated to use this in the future.
+  build-sign-publish-chainlink-plugins:
```

Comment:
This now represents a Chainlink core release, which is packaged with the latest plugins and published as such, on Chainlink core release tags.

This means we can not officially upgrade a set of plugins without releasing a new version of Chainlink core. These two releases (and their outputs) are ideally decoupled. 

@chainchad let's think next on a additional release channel which can compose a stable Chainlink img and the latests plugins. This can just be a separate tag and release pipeline from this same repo.

Reply:
And then we'd ideally have another internal one (release channel and repo), for teams to use for internal builds and dev/test on sandbox/staging.

Reply:
Good thoughts. Created DX-360 to capture this. We'd need to rejigger our `plugins/chainlink.Dockerfile` or create a new one that uses the core node as a base.